assets/js/hooks/session.js

```diff
@@ -109,6 +109,7 @@ const Session = {
 
     this.getElement("sections-list").addEventListener("click", (event) => {
       this.handleSectionsListClick(event);
+      this.handleSectionsListModuleClick(event);
```

Comment:
I'd put it in the same `handleSectionsListClick` handler, since it's about the whole sections panel :)